pre-docs/bolt_installing.md

```diff
@@ -225,6 +225,13 @@ The Puppet repository for the YUM package management system is [http://yum.puppe
         sudo dnf install puppet-bolt
         ```
 
+    -   Fedora 30
+
+        ```
+        sudo rpm -Uvh https://yum.puppet.com/puppet6/puppet6-release-fedora-30.noarch.rpm
```

Comment:
```suggestion
        sudo rpm -Uvh https://yum.puppet.com/puppet6-release-fedora-30.noarch.rpm
```

Reply:
Not sure if that is intentionally different than for the other fedora platforms. Lets confirm with RE before we update. Hopefully there is a standard location to point to for all the fedora platform versions.

Reply:
This is the standard location per RE. All release RPMs are in the top level directory.